fe/projects/AspNetCore/src/areas/Products/components/ProductDetail/ProductDetail.js

```diff
@@ -23,6 +28,9 @@ function ProductDetail(props) {
     const [canActiveModal, setCanActiveModal] = useState(true);
     const [showMore, setShowMore] = useState(false);
 
+    const [fabricsSliderOnStart, setFabricsSliderOnStart] = useState(true);
```

Comment:
**suggestion:** Is never used. if you want use, please change fabricsSliderOnStart to meanfuling name for product detail (not fabrics)